kount/client.py

```diff
@@ -30,73 +30,64 @@ class Client:
     raise_errors - False - log them only
                    True - raise them before request.post
     """
-    def __init__(self, url, key, timeout=5, raise_errors=RAISE_ERRORS):
+    def __init__(self, url, key, timeout=TIMEOUT, raise_errors=RAISE_ERRORS):
         self.url = url
-        self.timeout = timeout
-        self.kount_api_key = key
-        self.headers_api = {'X-Kount-Api-Key': self.kount_api_key}
-        self.xml_2_dict = XML_DICT
-        self.required = REQUIRED
-        self.notrequired = NOTREQUIRED
+        self._kount_api_key = key
         self.raise_errors = raise_errors
-        self.validator = RisValidator(raise_errors=self.raise_errors)
+        card_solted = Khash.hash_payment_token(token="666666669")
+        if card_solted != "6666662I8EDD7LNC77GP" and raise_errors:
```

Comment:
I think we discussed this check to be in the initialization of the `khash.py` class. The `client` object has nothing to do with knowing whether SALT phrase has been correctly configured.
Also, khashing `666666669` really results in `6666662I8EDD7LNC77GP` but this value **is not** provided by Kount team; they've given us the SHA of the SALT phrase instead and we again discussed using this value for checking. Please, stick to that common feature in the SDKs.